### PR TITLE
[HOPSWORKS-813] speedup TensorBoard in experiments service

### PR DESCRIPTION
**Pull request by robzor92**
## Make sure there is no duplicate PR for this issue

* **Please check if the PR meets the following requirements**
- [ ] Adds tests for the submitted changes (for bug fixes & features)
- [x] Passes the tests
- [x] HOPSWORKS JIRA issue has been opened for this PR
- [x] All commits have been squashed down to a single commit

* **Post a link to the associated JIRA issue**

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Comment by robzor92** [comment]
https://github.com/logicalclocks/hops-testing/pull/231

File: hopsworks-api/src/main/java/io/hops/hopsworks/api/tensorflow/TensorboardProxyServlet.java [hunk 1]
```diff
@@ -147,7 +147,7 @@ protected void service(HttpServletRequest servletRequest,
         targetUriObj = new URI(targetUri);
         targetUriHost = new URI(theHost);
       } catch (Exception e) {
-        throw new ServletException("Trying to process targetUri init parameter: ", e);
+        return;
```
Comment (tkakantousis) [inline hopsworks-api/src/main/java/io/hops/hopsworks/api/tensorflow/TensorboardProxyServlet.java]: @robzor92 add some logging here. If it's not an error that would indicate a problem with the backend, we can set it in FINE level.

File: hopsworks-api/src/main/java/io/hops/hopsworks/api/tensorflow/TensorboardProxyServlet.java [hunk 2]
```diff
@@ -207,7 +207,7 @@ protected void service(HttpServletRequest servletRequest,
...
```
Comment (tkakantousis) [inline hopsworks-api/src/main/java/io/hops/hopsworks/api/tensorflow/TensorboardProxyServlet.java]: @robzor92 add some logging here. If it's not an error that would indicate a problem with the backend, we can set it in FINE level.

Comment (robzor92) [reply hopsworks-api/src/main/java/io/hops/hopsworks/api/tensorflow/TensorboardProxyServlet.java]: done

File: hopsworks-api/src/main/java/io/hops/hopsworks/api/tensorflow/TensorboardProxyServlet.java (hunk 1 above)
Comment (robzor92) [reply hopsworks-api/src/main/java/io/hops/hopsworks/api/tensorflow/TensorboardProxyServlet.java]: done

**Pull request merged**